pytest-profiling/pytest_profiling.py

```diff
@@ -27,9 +27,10 @@ class Profiling(object):
     profs = []
     combined = None
 
-    def __init__(self, svg, dir):
+    def __init__(self, svg, dir, element_number):
```

Comment:
It may be good to make this a keyword argument with the current default to prevent a breaking change.